chart/keep/values.yaml

```
@@ -102,6 +102,7 @@ database:
   enabled: true
   replicaCount: 1
   size: 5Gi
+  storageClasss: deafult
```

@DmarshalTU 
is it default?
```suggestion
  storageClasss: default
```

---

thank to point it out, it should be empty